simulator/src/main/java/simblock/settings/SimulationConfiguration.java

```diff
@@ -193,4 +188,7 @@ public class SimulationConfiguration {
     0.72f,0.73f,0.74f,0.75f,0.76f,0.77f,0.78f,0.79f,0.8f,0.81f,0.82f,0.83f,0.84f,0.85f,0.86f,
     0.87f,0.88f,0.89f,0.9f,0.91f,0.92f,0.93f,0.94f,0.95f,0.96f
   };
+
+  public static int chia_k = 3;
```

Comment:
nit: follow this suggested naming convention:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
see the bottom of the table. 

chia_k -> CHIA_K

also make it final

`public static final int`

Reply:
Changed naming convention but kept it non-final because the algorithm specifies k can change. I don't think that will matter for our purposes, though, so I can make it final if you'd still like me to do so.